vm/src/vm/runners/builtin_runner/ec_op.rs

```diff
@@ -62,23 +60,22 @@ impl EcOpBuiltinRunner {
         m: &Felt252,
         height: u32,
     ) -> Result<(Felt252, Felt252), RunnerError> {
-        let mut slope = m.to_bigint();
+        let slope = m.to_biguint();
         let mut partial_sum_b = ProjectivePoint::from_affine(partial_sum.0, partial_sum.1)
             .map_err(|_| RunnerError::PointNotOnCurve(Box::new(partial_sum)))?;
         let mut doubled_point_b = ProjectivePoint::from_affine(doubled_point.0, doubled_point.1)
             .map_err(|_| RunnerError::PointNotOnCurve(Box::new(doubled_point)))?;
-        for _ in 0..height {
+        for i in 0..(Into::<u64>::into(height)).min(slope.bits()) {
```

Comment:
This would be more brief and equally valid:
```suggestion
        for i in 0..(height as u64).min(slope.bits()) {
```